inst/include/population_dynamics/fleet/fleet.hpp

```diff
@@ -57,12 +40,20 @@ struct Fleet : public FIMSObject<Type> {
   int selectivity_id = -999; /*!< id of selectivity component*/
   std::shared_ptr<fims::SelectivityBase<Type>>
       selectivity; /*!< selectivity component*/
+  
+  int observed_index_data_id = -999; /*!< id of index data */
+  std::shared_ptr<fims::DataObject<Type>> observed_index_data; /*!< observed index data*/
+
+  int observed_agecomp_data_id = -999; /*!< id of age comp data */
+  std::shared_ptr<fims::DataObject<Type>> 
+  observed_agecomp_data; /*!< observed agecomp data*/
 
   // Mortality and catchability
-  std::vector<Type> log_Fmort; /*!< estimated parameter: log Fishing mortality*/
-  std::vector<Type>
+  ParameterVector log_Fmort; /*!< estimated parameter: log Fishing mortality*/
+  ParameterVector
       log_q; /*!< estimated parameter: catchability of the fleet */
 
+  Type log_obs_error; /*!< estimated parameter: observation error associated with index */
```

Comment:
Does log_obs_error need to be ParameterVector if it is an estimated parameter?

Reply:
Good question - I was only using ParameterVector for vectors of parameters, and I thought we could use Type for scalar parameters since when they are populated they are given type Parameter? but I'm not 100% sure